test/jdk/java/lang/ClassLoader/loadLibraryDeadlock/TestLoadLibraryDeadlock.java

```diff
@@ -108,51 +107,13 @@ private static OutputAnalyzer signJar(String jarToSign) throws Throwable {
         );
     }
 
-    private static Process runJavaCommand(String... command) throws Throwable {
+    private static OutputAnalyzer runJavaCommand(String... command) throws Throwable {
         String java = JDKToolFinder.getJDKTool("java");
         List<String> commands = new ArrayList<>();
         Collections.addAll(commands, java);
+        Collections.addAll(commands, Utils.getTestJavaOpts());
         Collections.addAll(commands, command);
```

Comment:
The prose description talks about using ProcessTools, but the runCommand code doesn't use ProcessTools.createTestJavaProcessBuilder. It could save a few steps.

Reply:
`runCommand` sets the working directory whereas `ProcessTools.createTestJavaProcessBuilder` does not.  Either way is okay with me if we want to use `createTestJavaProcessBuilder` (which will set the VM options).

```
+        ProcessBuilder pb = ProcessTools.createTestJavaProcessBuilder("-cp",
+                                                                      "a.jar" + classPathSeparator +
+                                                                      "b.jar" + classPathSeparator +
+                                                                      "c.jar",
+                                                                      "-Djava.library.path=" + testLibraryPath,
+                                                                      "LoadLibraryDeadlock")
+                                        .directory(new File(testClassPath));
+        OutputAnalyzer outputAnalyzer = ProcessTools.executeCommand(pb)
+                .shouldHaveExitValue(0);
```

or set the class path to the explicit path of the JAR files.

Reply:
Updated to use `createTestJavaProcessBuilder` and more clean up.